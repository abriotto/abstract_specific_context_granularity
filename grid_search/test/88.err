  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  3%|#####6                                                                                                                                                                                      | 3/99 [00:00<00:04, 23.56it/s]  6%|###########3                                                                                                                                                                                | 6/99 [00:00<00:05, 15.65it/s]  8%|###############1                                                                                                                                                                            | 8/99 [00:00<00:05, 15.90it/s] 10%|##################8                                                                                                                                                                        | 10/99 [00:00<00:05, 16.68it/s] 12%|######################6                                                                                                                                                                    | 12/99 [00:00<00:07, 11.85it/s] 14%|##########################4                                                                                                                                                                | 14/99 [00:00<00:06, 13.36it/s] 16%|##############################2                                                                                                                                                            | 16/99 [00:01<00:06, 12.42it/s] 18%|##################################                                                                                                                                                         | 18/99 [00:01<00:07, 10.58it/s] 20%|#####################################7                                                                                                                                                     | 20/99 [00:01<00:08,  9.65it/s] 22%|#########################################5                                                                                                                                                 | 22/99 [00:01<00:08,  9.62it/s] 24%|#############################################3                                                                                                                                             | 24/99 [00:02<00:08,  8.57it/s] 25%|###############################################2                                                                                                                                           | 25/99 [00:02<00:09,  7.76it/s] 26%|#################################################1                                                                                                                                         | 26/99 [00:02<00:10,  7.24it/s] 27%|##################################################9                                                                                                                                        | 27/99 [00:02<00:10,  6.98it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:02<00:08,  8.70it/s] 30%|########################################################6                                                                                                                                  | 30/99 [00:02<00:08,  8.15it/s] 31%|##########################################################5                                                                                                                                | 31/99 [00:03<00:09,  7.43it/s] 32%|############################################################4                                                                                                                              | 32/99 [00:03<00:09,  7.11it/s] 34%|################################################################2                                                                                                                          | 34/99 [00:03<00:08,  7.52it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:03<00:09,  6.99it/s] 36%|####################################################################                                                                                                                       | 36/99 [00:03<00:09,  6.91it/s] 38%|#######################################################################7                                                                                                                   | 38/99 [00:04<00:07,  8.53it/s] 39%|#########################################################################6                                                                                                                 | 39/99 [00:04<00:07,  7.92it/s] 41%|#############################################################################4                                                                                                             | 41/99 [00:04<00:07,  7.82it/s] 42%|###############################################################################3                                                                                                           | 42/99 [00:04<00:07,  7.26it/s] 44%|###################################################################################1                                                                                                       | 44/99 [00:04<00:05,  9.45it/s] 46%|######################################################################################8                                                                                                    | 46/99 [00:05<00:06,  8.10it/s] 47%|########################################################################################7                                                                                                  | 47/99 [00:05<00:06,  7.49it/s] 48%|##########################################################################################6                                                                                                | 48/99 [00:05<00:07,  7.09it/s] 51%|##############################################################################################4                                                                                            | 50/99 [00:05<00:06,  7.57it/s] 54%|####################################################################################################1                                                                                      | 53/99 [00:05<00:04, 11.00it/s] 56%|#######################################################################################################8                                                                                   | 55/99 [00:05<00:04,  9.86it/s] 58%|###########################################################################################################6                                                                               | 57/99 [00:06<00:03, 10.86it/s] 60%|###############################################################################################################4                                                                           | 59/99 [00:06<00:03, 10.34it/s] 63%|#####################################################################################################################1                                                                     | 62/99 [00:06<00:02, 12.87it/s] 65%|########################################################################################################################8                                                                  | 64/99 [00:06<00:03, 11.28it/s] 67%|############################################################################################################################6                                                              | 66/99 [00:06<00:03, 10.15it/s] 69%|################################################################################################################################4                                                          | 68/99 [00:07<00:03,  9.61it/s] 71%|####################################################################################################################################2                                                      | 70/99 [00:07<00:03,  9.30it/s] 72%|######################################################################################################################################1                                                    | 71/99 [00:07<00:03,  8.41it/s] 74%|#########################################################################################################################################8                                                 | 73/99 [00:07<00:03,  8.35it/s] 75%|###########################################################################################################################################7                                               | 74/99 [00:08<00:03,  7.81it/s] 77%|###############################################################################################################################################5                                           | 76/99 [00:08<00:02,  9.29it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:08<00:02,  8.26it/s] 79%|###################################################################################################################################################3                                       | 78/99 [00:08<00:02,  7.59it/s] 81%|#######################################################################################################################################################1                                   | 80/99 [00:08<00:01,  9.54it/s] 83%|##########################################################################################################################################################8                                | 82/99 [00:08<00:01,  9.06it/s] 84%|############################################################################################################################################################7                              | 83/99 [00:09<00:01,  8.21it/s] 85%|##############################################################################################################################################################6                            | 84/99 [00:09<00:01,  7.71it/s] 86%|################################################################################################################################################################5                          | 85/99 [00:09<00:01,  7.32it/s] 87%|##################################################################################################################################################################4                        | 86/99 [00:09<00:01,  6.75it/s] 88%|####################################################################################################################################################################3                      | 87/99 [00:09<00:01,  6.62it/s] 89%|######################################################################################################################################################################2                    | 88/99 [00:09<00:01,  6.33it/s] 91%|##########################################################################################################################################################################                 | 90/99 [00:10<00:01,  7.18it/s] 92%|###########################################################################################################################################################################8               | 91/99 [00:10<00:01,  6.97it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:10<00:00,  8.55it/s] 96%|###################################################################################################################################################################################4       | 95/99 [00:10<00:00,  8.54it/s] 97%|#####################################################################################################################################################################################3     | 96/99 [00:10<00:00,  7.77it/s] 99%|#########################################################################################################################################################################################1 | 98/99 [00:11<00:00,  8.03it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  7.60it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  8.82it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s]  4%|#######5                                                                                                                                                                                    | 1/25 [00:00<00:04,  5.94it/s] 12%|######################5                                                                                                                                                                     | 3/25 [00:00<00:02,  8.82it/s] 16%|##############################                                                                                                                                                              | 4/25 [00:00<00:02,  7.69it/s] 24%|#############################################1                                                                                                                                              | 6/25 [00:00<00:02,  7.91it/s] 28%|####################################################6                                                                                                                                       | 7/25 [00:00<00:02,  7.43it/s] 32%|############################################################1                                                                                                                               | 8/25 [00:01<00:02,  7.06it/s] 40%|##########################################################################8                                                                                                                | 10/25 [00:01<00:01,  7.53it/s] 44%|##################################################################################2                                                                                                        | 11/25 [00:01<00:01,  7.21it/s] 52%|#################################################################################################2                                                                                         | 13/25 [00:01<00:01,  8.63it/s] 64%|#######################################################################################################################6                                                                   | 16/25 [00:01<00:00,  9.62it/s] 72%|######################################################################################################################################6                                                    | 18/25 [00:02<00:00, 10.54it/s] 80%|#####################################################################################################################################################6                                     | 20/25 [00:02<00:00, 10.25it/s] 88%|####################################################################################################################################################################5                      | 22/25 [00:02<00:00, 11.90it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:02<00:00,  9.21it/s]100%|###########################################################################################################################################################################################| 25/25 [00:02<00:00,  8.96it/s]
