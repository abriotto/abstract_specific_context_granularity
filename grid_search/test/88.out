# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.99", "--temperature=1.5"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6798633337020874, "acc": 0.557178258895874, "length": 3.1787524223327637, "mode": "train", "epoch": 1}
{"loss": 0.6146536469459534, "acc": 0.6473598480224609, "length": 4.0, "mode": "test", "epoch": 1}
{"loss": 0.5583251118659973, "acc": 0.7047290205955505, "length": 3.886624574661255, "mode": "train", "epoch": 2}
{"loss": 0.5376870632171631, "acc": 0.7422442436218262, "length": 4.0, "mode": "test", "epoch": 2}
{"loss": 0.4592452645301819, "acc": 0.790385365486145, "length": 3.9764304161071777, "mode": "train", "epoch": 3}
{"loss": 0.486865758895874, "acc": 0.7808580994606018, "length": 4.0, "mode": "test", "epoch": 3}
{"loss": 0.4045957922935486, "acc": 0.8200069665908813, "length": 3.991428852081299, "mode": "train", "epoch": 4}
{"loss": 0.42648372054100037, "acc": 0.8150163888931274, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.36482393741607666, "acc": 0.8415802717208862, "length": 3.9957945346832275, "mode": "train", "epoch": 5}
{"loss": 0.4489877223968506, "acc": 0.7886139154434204, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.34068217873573303, "acc": 0.8540027141571045, "length": 3.9976067543029785, "mode": "train", "epoch": 6}
{"loss": 0.4066619277000427, "acc": 0.8287127614021301, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.323725163936615, "acc": 0.8630858063697815, "length": 3.9981374740600586, "mode": "train", "epoch": 7}
{"loss": 0.38810545206069946, "acc": 0.8316006064414978, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.3003609776496887, "acc": 0.8779139518737793, "length": 3.998488426208496, "mode": "train", "epoch": 8}
{"loss": 0.3521686792373657, "acc": 0.8523927927017212, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.28024742007255554, "acc": 0.8863011002540588, "length": 3.9989256858825684, "mode": "train", "epoch": 9}
{"loss": 0.35680776834487915, "acc": 0.8537954092025757, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.2652589976787567, "acc": 0.896308183670044, "length": 3.999096155166626, "mode": "train", "epoch": 10}
{"loss": 0.3279682397842407, "acc": 0.8669968247413635, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.24660076200962067, "acc": 0.9048025012016296, "length": 3.9992427825927734, "mode": "train", "epoch": 11}
{"loss": 0.3090388774871826, "acc": 0.8830032348632812, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.236927792429924, "acc": 0.9115427732467651, "length": 3.9988808631896973, "mode": "train", "epoch": 12}
{"loss": 0.3106493055820465, "acc": 0.8794553875923157, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.22228115797042847, "acc": 0.9163721203804016, "length": 3.99906325340271, "mode": "train", "epoch": 13}
{"loss": 0.29531359672546387, "acc": 0.8816006779670715, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.21056948602199554, "acc": 0.923902153968811, "length": 3.9993362426757812, "mode": "train", "epoch": 14}
{"loss": 0.27219128608703613, "acc": 0.8917491436004639, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.1980055570602417, "acc": 0.9264113306999207, "length": 3.999336004257202, "mode": "train", "epoch": 15}
{"loss": 0.2684873640537262, "acc": 0.8934818506240845, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.18732190132141113, "acc": 0.931759774684906, "length": 3.9993996620178223, "mode": "train", "epoch": 16}
{"loss": 0.23865704238414764, "acc": 0.9110560417175293, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.18295986950397491, "acc": 0.9335827231407166, "length": 3.9995076656341553, "mode": "train", "epoch": 17}
{"loss": 0.25089162588119507, "acc": 0.9086635112762451, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.17621712386608124, "acc": 0.9363129138946533, "length": 3.999566078186035, "mode": "train", "epoch": 18}
{"loss": 0.2610858976840973, "acc": 0.9037954211235046, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.1610795259475708, "acc": 0.9424635767936707, "length": 3.9997060298919678, "mode": "train", "epoch": 19}
{"loss": 0.2206047922372818, "acc": 0.9219472408294678, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.15761704742908478, "acc": 0.9439533352851868, "length": 3.99973726272583, "mode": "train", "epoch": 20}
{"loss": 0.21943572163581848, "acc": 0.9184818267822266, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.14769592881202698, "acc": 0.9508616924285889, "length": 3.9996116161346436, "mode": "train", "epoch": 21}
{"loss": 0.20815201103687286, "acc": 0.9287127256393433, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.13941983878612518, "acc": 0.955637514591217, "length": 3.999690294265747, "mode": "train", "epoch": 22}
{"loss": 0.2015543133020401, "acc": 0.9357259273529053, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.13720208406448364, "acc": 0.9528568983078003, "length": 3.9997730255126953, "mode": "train", "epoch": 23}
{"loss": 0.2192155420780182, "acc": 0.9305282235145569, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.1302267611026764, "acc": 0.9585902094841003, "length": 3.999819755554199, "mode": "train", "epoch": 24}
{"loss": 0.20000359416007996, "acc": 0.9313530921936035, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.1267326921224594, "acc": 0.9578500390052795, "length": 3.9998676776885986, "mode": "train", "epoch": 25}
{"loss": 0.18354646861553192, "acc": 0.9322609305381775, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.12093272060155869, "acc": 0.9623423218727112, "length": 3.9998767375946045, "mode": "train", "epoch": 26}
{"loss": 0.20780715346336365, "acc": 0.9278878569602966, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.120126873254776, "acc": 0.9618732333183289, "length": 3.999842643737793, "mode": "train", "epoch": 27}
{"loss": 0.19220559298992157, "acc": 0.9363860487937927, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.11623129993677139, "acc": 0.9639115929603577, "length": 3.999842882156372, "mode": "train", "epoch": 28}
{"loss": 0.17790672183036804, "acc": 0.9434818625450134, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.11413396894931793, "acc": 0.9629481434822083, "length": 3.999859571456909, "mode": "train", "epoch": 29}
{"loss": 0.1758807748556137, "acc": 0.9424092173576355, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.10722345858812332, "acc": 0.9665617942810059, "length": 3.999922037124634, "mode": "train", "epoch": 30}
{"loss": 0.17116686701774597, "acc": 0.945462167263031, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.10273710638284683, "acc": 0.9688211679458618, "length": 3.9999167919158936, "mode": "train", "epoch": 31}
{"loss": 0.17653533816337585, "acc": 0.9429867267608643, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.09823967516422272, "acc": 0.9699798822402954, "length": 3.999924421310425, "mode": "train", "epoch": 32}
{"loss": 0.15623100101947784, "acc": 0.9501650929450989, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.09466561675071716, "acc": 0.9700897932052612, "length": 3.9999189376831055, "mode": "train", "epoch": 33}
{"loss": 0.16639110445976257, "acc": 0.9464521408081055, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.09135493636131287, "acc": 0.9721848368644714, "length": 3.9999380111694336, "mode": "train", "epoch": 34}
{"loss": 0.15493139624595642, "acc": 0.9485971927642822, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.09521613270044327, "acc": 0.9703649282455444, "length": 3.999922037124634, "mode": "train", "epoch": 35}
{"loss": 0.17358754575252533, "acc": 0.9511551260948181, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.10033337026834488, "acc": 0.9679883718490601, "length": 3.9997711181640625, "mode": "train", "epoch": 36}
{"loss": 0.1870701014995575, "acc": 0.9430693984031677, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.08828210085630417, "acc": 0.9726808667182922, "length": 3.999931812286377, "mode": "train", "epoch": 37}
{"loss": 0.1695978343486786, "acc": 0.9477723240852356, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.09043804556131363, "acc": 0.9716073870658875, "length": 3.999948501586914, "mode": "train", "epoch": 38}
{"loss": 0.1548505276441574, "acc": 0.9537953734397888, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.09748820960521698, "acc": 0.9680795669555664, "length": 3.9999520778656006, "mode": "train", "epoch": 39}
{"loss": 0.17249152064323425, "acc": 0.9522277116775513, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.08918751031160355, "acc": 0.9718815684318542, "length": 3.999922752380371, "mode": "train", "epoch": 40}
{"loss": 0.166143536567688, "acc": 0.9461221694946289, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.07969128340482712, "acc": 0.9761273264884949, "length": 3.999946117401123, "mode": "train", "epoch": 41}
{"loss": 0.13769686222076416, "acc": 0.9581682682037354, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.0800342783331871, "acc": 0.9759067296981812, "length": 3.999939441680908, "mode": "train", "epoch": 42}
{"loss": 0.14636875689029694, "acc": 0.9556106328964233, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.07931683212518692, "acc": 0.9751607179641724, "length": 3.9999122619628906, "mode": "train", "epoch": 43}
{"loss": 0.14977630972862244, "acc": 0.9614684581756592, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.07763475179672241, "acc": 0.9769546389579773, "length": 3.9999606609344482, "mode": "train", "epoch": 44}
{"loss": 0.13010460138320923, "acc": 0.9629536867141724, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.07373745739459991, "acc": 0.9771208167076111, "length": 3.999974489212036, "mode": "train", "epoch": 45}
{"loss": 0.1367688775062561, "acc": 0.9629536867141724, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.0734887570142746, "acc": 0.9783889055252075, "length": 3.9999775886535645, "mode": "train", "epoch": 46}
{"loss": 0.12642432749271393, "acc": 0.9650166034698486, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.06312013417482376, "acc": 0.9811713099479675, "length": 3.999911069869995, "mode": "train", "epoch": 47}
{"loss": 0.13026607036590576, "acc": 0.9639439582824707, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.07146364450454712, "acc": 0.9787440299987793, "length": 3.9999263286590576, "mode": "train", "epoch": 48}
{"loss": 0.13837407529354095, "acc": 0.9676567316055298, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.06449299305677414, "acc": 0.9819999933242798, "length": 3.9999821186065674, "mode": "train", "epoch": 49}
{"loss": 0.13783790171146393, "acc": 0.9632012248039246, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.06679824739694595, "acc": 0.9799607396125793, "length": 3.999985456466675, "mode": "train", "epoch": 50}
{"loss": 0.15093252062797546, "acc": 0.9640263319015503, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.07808215171098709, "acc": 0.9765982031822205, "length": 3.9999914169311523, "mode": "train", "epoch": 51}
{"loss": 0.14237965643405914, "acc": 0.9584982991218567, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.07745584100484848, "acc": 0.9758539795875549, "length": 3.9999911785125732, "mode": "train", "epoch": 52}
{"loss": 0.11774272471666336, "acc": 0.9647690057754517, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.07068196684122086, "acc": 0.9797956943511963, "length": 3.999992847442627, "mode": "train", "epoch": 53}
{"loss": 0.11891566962003708, "acc": 0.9679043889045715, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.05756807699799538, "acc": 0.9841783046722412, "length": 3.9999940395355225, "mode": "train", "epoch": 54}
{"loss": 0.1140480563044548, "acc": 0.9685643911361694, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.061250001192092896, "acc": 0.9823590517044067, "length": 3.9999942779541016, "mode": "train", "epoch": 55}
{"loss": 0.12063392996788025, "acc": 0.9649339914321899, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.059829872101545334, "acc": 0.9827451109886169, "length": 3.999997854232788, "mode": "train", "epoch": 56}
{"loss": 0.12370628863573074, "acc": 0.965676486492157, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.06195170059800148, "acc": 0.9830483794212341, "length": 3.999998092651367, "mode": "train", "epoch": 57}
{"loss": 0.1515551060438156, "acc": 0.9607262015342712, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.05267792567610741, "acc": 0.9863284230232239, "length": 3.99999737739563, "mode": "train", "epoch": 58}
{"loss": 0.14064083993434906, "acc": 0.9654290080070496, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.048893287777900696, "acc": 0.986714243888855, "length": 3.9999942779541016, "mode": "train", "epoch": 59}
{"loss": 0.11468548327684402, "acc": 0.9705445766448975, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.04755941405892372, "acc": 0.98751300573349, "length": 3.9999895095825195, "mode": "train", "epoch": 60}
{"loss": 0.09540747106075287, "acc": 0.9734323024749756, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.04921010881662369, "acc": 0.9870726466178894, "length": 3.9999988079071045, "mode": "train", "epoch": 61}
{"loss": 0.1319497674703598, "acc": 0.9669142365455627, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.051906779408454895, "acc": 0.9862733483314514, "length": 3.9999992847442627, "mode": "train", "epoch": 62}
{"loss": 0.12460007518529892, "acc": 0.9618811011314392, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.05583969131112099, "acc": 0.9842321276664734, "length": 3.9999701976776123, "mode": "train", "epoch": 63}
{"loss": 0.14442431926727295, "acc": 0.9575907588005066, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.07446277886629105, "acc": 0.9778942465782166, "length": 3.9999992847442627, "mode": "train", "epoch": 64}
{"loss": 0.15360979735851288, "acc": 0.9601485729217529, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.07215674966573715, "acc": 0.9769845604896545, "length": 3.999997854232788, "mode": "train", "epoch": 65}
{"loss": 0.13848073780536652, "acc": 0.9602309465408325, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.06647489219903946, "acc": 0.9791610240936279, "length": 3.9999420642852783, "mode": "train", "epoch": 66}
{"loss": 0.11784444749355316, "acc": 0.970544695854187, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.056947581470012665, "acc": 0.9851431250572205, "length": 3.9999985694885254, "mode": "train", "epoch": 67}
{"loss": 0.12309692054986954, "acc": 0.9665842652320862, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.0694369301199913, "acc": 0.9797406792640686, "length": 3.999995231628418, "mode": "train", "epoch": 68}
{"loss": 0.11619028449058533, "acc": 0.9709570407867432, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.07594530284404755, "acc": 0.9776726961135864, "length": 3.999983787536621, "mode": "train", "epoch": 69}
{"loss": 0.16194675862789154, "acc": 0.9527227282524109, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.09150251001119614, "acc": 0.9713340401649475, "length": 3.99999737739563, "mode": "train", "epoch": 70}
{"loss": 0.14921733736991882, "acc": 0.9570958018302917, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.0791921615600586, "acc": 0.977260172367096, "length": 3.9999983310699463, "mode": "train", "epoch": 71}
{"loss": 0.10727940499782562, "acc": 0.9702970385551453, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.05666585639119148, "acc": 0.9834340810775757, "length": 3.999998092651367, "mode": "train", "epoch": 72}
{"loss": 0.09309235960245132, "acc": 0.9768152832984924, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.05197490379214287, "acc": 0.9853636622428894, "length": 3.99999737739563, "mode": "train", "epoch": 73}
{"loss": 0.12419945746660233, "acc": 0.967491626739502, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.051554106175899506, "acc": 0.98514324426651, "length": 3.9999985694885254, "mode": "train", "epoch": 74}
{"loss": 0.10458291321992874, "acc": 0.9751651287078857, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.04141177982091904, "acc": 0.9895533919334412, "length": 3.999997854232788, "mode": "train", "epoch": 75}
{"loss": 0.09606774151325226, "acc": 0.9733498692512512, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.03812859207391739, "acc": 0.9901319146156311, "length": 3.9999938011169434, "mode": "train", "epoch": 76}
{"loss": 0.08746489882469177, "acc": 0.9771453738212585, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.04033719003200531, "acc": 0.9890846014022827, "length": 3.9999918937683105, "mode": "train", "epoch": 77}
{"loss": 0.08846776932477951, "acc": 0.9772278666496277, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.03361125290393829, "acc": 0.9918962717056274, "length": 3.9999988079071045, "mode": "train", "epoch": 78}
{"loss": 0.08706636726856232, "acc": 0.979290246963501, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.05645915865898132, "acc": 0.9851157069206238, "length": 3.999999523162842, "mode": "train", "epoch": 79}
{"loss": 0.12539000809192657, "acc": 0.9648514986038208, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.0504378005862236, "acc": 0.9852532744407654, "length": 3.999992847442627, "mode": "train", "epoch": 80}
{"loss": 0.09815993160009384, "acc": 0.9766501188278198, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.04361081123352051, "acc": 0.9879546165466309, "length": 3.999995231628418, "mode": "train", "epoch": 81}
{"loss": 0.0888923928141594, "acc": 0.9785476326942444, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.04798370227217674, "acc": 0.9870726466178894, "length": 3.9999964237213135, "mode": "train", "epoch": 82}
{"loss": 0.0936862975358963, "acc": 0.976732611656189, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.05226799473166466, "acc": 0.9853912591934204, "length": 3.999997854232788, "mode": "train", "epoch": 83}
{"loss": 0.08115103095769882, "acc": 0.9768975973129272, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.04552730545401573, "acc": 0.9877892732620239, "length": 3.999999761581421, "mode": "train", "epoch": 84}
{"loss": 0.08488284051418304, "acc": 0.9774752259254456, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.04320961982011795, "acc": 0.9876788258552551, "length": 3.9999959468841553, "mode": "train", "epoch": 85}
{"loss": 0.1050962433218956, "acc": 0.9734323620796204, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.04960086941719055, "acc": 0.986025333404541, "length": 4.0, "mode": "train", "epoch": 86}
{"loss": 0.08946536481380463, "acc": 0.975659966468811, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.041985806077718735, "acc": 0.98856121301651, "length": 3.999999761581421, "mode": "train", "epoch": 87}
{"loss": 0.0973033457994461, "acc": 0.9721947312355042, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.048209838569164276, "acc": 0.9858047962188721, "length": 3.9999985694885254, "mode": "train", "epoch": 88}
{"loss": 0.08448994904756546, "acc": 0.9754949808120728, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.04160018637776375, "acc": 0.9890849590301514, "length": 3.9999988079071045, "mode": "train", "epoch": 89}
{"loss": 0.08882779628038406, "acc": 0.9768151044845581, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.04393638297915459, "acc": 0.9864388704299927, "length": 4.0, "mode": "train", "epoch": 90}
{"loss": 0.13069793581962585, "acc": 0.9669967293739319, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.049071021378040314, "acc": 0.985968828201294, "length": 3.9999845027923584, "mode": "train", "epoch": 91}
{"loss": 0.08419464528560638, "acc": 0.9768975973129272, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.0421944260597229, "acc": 0.9870548248291016, "length": 3.9995429515838623, "mode": "train", "epoch": 92}
{"loss": 0.10280224680900574, "acc": 0.969059407711029, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.049652911722660065, "acc": 0.9852261543273926, "length": 4.0, "mode": "train", "epoch": 93}
{"loss": 0.09047164022922516, "acc": 0.9754123091697693, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.0472143292427063, "acc": 0.9867143034934998, "length": 4.0, "mode": "train", "epoch": 94}
{"loss": 0.09882349520921707, "acc": 0.9691419005393982, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.06269056349992752, "acc": 0.9815875887870789, "length": 3.9999983310699463, "mode": "train", "epoch": 95}
{"loss": 0.09871821105480194, "acc": 0.9731848239898682, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.06141182407736778, "acc": 0.9815324544906616, "length": 3.999999523162842, "mode": "train", "epoch": 96}
{"loss": 0.11732110381126404, "acc": 0.9642738699913025, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.0547272190451622, "acc": 0.9832967519760132, "length": 4.0, "mode": "train", "epoch": 97}
{"loss": 0.11226364225149155, "acc": 0.9629538655281067, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.05806238204240799, "acc": 0.9825798869132996, "length": 4.0, "mode": "train", "epoch": 98}
{"loss": 0.12223975360393524, "acc": 0.9677392840385437, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.04873966798186302, "acc": 0.9858047962188721, "length": 3.9999988079071045, "mode": "train", "epoch": 99}
{"loss": 0.10845600813627243, "acc": 0.9698845148086548, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.056028518825769424, "acc": 0.9835723042488098, "length": 3.999999761581421, "mode": "train", "epoch": 100}
{"loss": 0.10945001244544983, "acc": 0.9694718718528748, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.689568817615509
